### ISSUE TITLE
没有在代码中看到关于GAF函数的使用

### ISSUE THREAD

**Issue by 2238043627** [yanghangAI/PRIS#5]
论文中写道提出了一个梯度近似函数（GAF），但是并没有看到训练模型的代码中使用到这个函数

**Comment by yanghangAI** [comment]
在util.py第29行
def round_diff(x):
    sign = torch.ones_like(x)
    sign[torch.floor(x) % 2 == 0] = -1
    y = sign * torch.cos(x * torch.pi) / 2
    out = torch.round(x) + y - y.detach()
    return out
